__tests__/unit-tests.spec.ts

```diff
@@ -304,5 +355,56 @@ describe('@typeofweb/schema unit tests', () => {
         '{ a: ≫string≪, b: ≫number≪, "no elo koleś": ≫boolean≪, c: { e: ≫(≫string[]≪ | { xxx: ≫number≪ })[]≪ } }',
       );
     });
+
+    it('should work for simple validators with modifiers', () => {
+      expect(schemaToString(optional(string()))).toEqual('(≫string≪ | undefined)');
+      expect(schemaToString(nullable(string()))).toEqual('(≫string≪ | null)');
+      expect(schemaToString(optional(number()))).toEqual('(≫number≪ | undefined)');
+      expect(schemaToString(nullable(number()))).toEqual('(≫number≪ | null)');
+      expect(schemaToString(optional(boolean()))).toEqual('(≫boolean≪ | undefined)');
+      expect(schemaToString(nullable(boolean()))).toEqual('(≫boolean≪ | null)');
+      expect(schemaToString(optional(date()))).toEqual('(≫Date≪ | undefined)');
+      expect(schemaToString(nullable(date()))).toEqual('(≫Date≪ | null)');
```

Comment:
Zastanawia mnie czy nie lepiej rozbić to na pojedyncze testy?

Reply:
Po co?

Reply:
Użycie jednej asercji na test jest dobrą praktyką.

Reply:
Wydaje mi się, że zbyt literalnie odczytujesz tę zasadę. IMHO dobrą praktyką jest testowanie jednej rzeczy na test, ale do przetestowania jednej rzeczy można użyć wielu asercji.